cosmos-sdk-proto/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cosmos-sdk-proto"
-version = "0.7.0" # Also update html_root_url in lib.rs when bumping this
+version = "0.7.1" # Also update html_root_url in lib.rs when bumping this
```

Comment:
If we do that, perhaps it's worth considering trying to (temporarily) switch to Informal's fork of Prost? If nothing else, it would allow the docs to build /cc @xla
https://github.com/informalsystems/tendermint-rs/pull/1005/files